CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_marketo v0.13.0
+[PR #43](https://github.com/fivetran/dbt_marketo/pull/43) includes the following updates:
+
+## Breaking Change
+- Updates the default configuration for the `marketo__enable_campaigns` and `marketo__enable_programs` variables from disabled to enabled.
+
+- Each variable will enable the following models by default:
```

Comment:
This isn't capturing all the changes. For example, enabling `marketo__enable_campaigns` adds the `campaign_type` and `program_id` in the `marketo__email_sends` model. Also it would be good to add in here the upstream staging models these enable as well. Especially since Quickstart users will only see this changelog not and not the upstream one.

Reply:
Added a more in-depth changelog section!